index.bs

```diff
@@ -172,7 +170,9 @@ The {{PerformanceElementTiming/element}} attribute's getter must run the [=get a
 
 Note: This means that an element that is no longer <a>descendant</a> of the {{Document}} will no longer be returned by {{PerformanceElementTiming/element}}'s attribute getter.
 
-The {{PerformanceElementTiming/url}} attribute's getter must return <a>request</a>'s <a>resolved URL</a>.
+The {{PerformanceElementTiming/url}} attribute's getter must return <a>request</a>'s <a>resolved URL</a>, trimmed to the first 100 characters if it is a data URL [[RFC2397]].
```

Comment:
Maybe make this a bit more algorithmy, and link this to the URL's [scheme](https://url.spec.whatwg.org/#concept-url-scheme) being "data"?

Reply:
Done.